pom.xml

```diff
@@ -168,6 +168,26 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <version>5.0.0</version>
```

Comment:
How did you test this locally?  The latest version of git-commit-id-plugin is 4.9.10: https://mvnrepository.com/artifact/pl.project13.maven/git-commit-id-plugin while the latest version of git-commit-id-maven-plugin is 9.0.2: git-commit-id-maven-plugin